dotnet/HtmlFormatter/Cucumber.HtmlFormatter.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>10.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
+    <Deterministic Condition="'$(Configuration)' == 'Release'">false</Deterministic>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>Cucumber.HtmlFormatter.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Version">
+    <VersionNumber>21.9.0</VersionNumber>
+    <Version Condition="'$(SnapshotSuffix)' != ''">$(VersionNumber)-$(SnapshotSuffix)</Version>
+    <Version Condition="'$(SnapshotSuffix)' == ''">$(VersionNumber)</Version>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <None Remove="Resources\index.mustache.html" />
+    <None Remove="Resources\main.css" />
+    <None Remove="Resources\main.js" />
+    <None Remove="Resources\main.js.LICENSE.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\index.mustache.html" />
```

Comment:
You can use a glob based adding, so we don't have to keep the project file in sync with the resources:
```
<EmbeddedResource Include="Resources\**\*" />
```

Reply:
Adopted